src/components/test/Calculator.test.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```

Comment:
- [ ] kindly consider making sure that all your tests are passing  as they should to full-fill the project requirement cause currently all your tests are failing

Reply:
Hi @divinecharlotte i don't understand this, all the test are passing on my end, i don't get it when you say they are all failing

Reply:
please recheck

Reply:
![Uploading image.png…]()

Reply:
hey @Tracy-miranja  here is the image below 👇 
<img width="1020" alt="Screen Shot 2023-04-06 at 23 31 53" src="https://user-images.githubusercontent.com/60146030/230498009-e7c24e10-daae-4e5c-8be3-538fa73ac8cd.png">